gym/vector/async_vector_env.py

```diff
@@ -635,6 +635,16 @@ def _worker(index, env_fn, pipe, parent_pipe, shared_memory, error_queue):
     assert shared_memory is None
     env = env_fn()
     parent_pipe.close()
+
+    def step_fn(actions):
```

Comment:
Why are we defining this function? It doesn't seem to be called anywhere. If it's actually used, it'd need a type hint and some comments/docstring

Reply:
Fixed in fcebd76b000980bac0d261b5bff1d6ecc48645ab

Reply:
Oops. Actually fixed in 15208e3c67c89df95792e2291bd9b62107e838a1